src/goTo.ts

```diff
@@ -148,9 +148,7 @@ export class GoTo implements vscode.DefinitionProvider {
       title: 'Bust-A-Gem ripping...',
     };
 
-    await vscode.window.withProgress(progressOptions, async () => {
-      await util.exec(cmd, { cwd: this.rootPath });
-    });
+    vscode.window.withProgress(progressOptions, () => util.exec(cmd, { cwd: this.rootPath }));
```

Comment:
The whole idea here is that we don't `await` for this command to finish. We just let it run in the background. I also checked that "Bust-A-Gem ripping..." message goes away more-less when the ripping finishes. I haven't looked into how that works in the extension, I just tested it as a black-box, that it works.